app.js

```diff
@@ -1,11 +1,119 @@
+    link: 'https://alucardsanin.github.io/Portfolio/',
+    source: 'https://github.com/AlucardSanin/Portfolio',
+    class: 'card-n',
+    },
+  {
+    id: 4,
+    title: 'Ironman',    
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    tags: ['css', 'html', 'bootsrap', 'Ruby'],
+    link: 'https://alucardsanin.github.io/Portfolio/',
+    source: 'https://github.com/AlucardSanin/Portfolio',
+    class: 'card-nr',
+  },
+];
+
+const body = document.querySelector('body');
+const Grid = document.createElement('div');
+Grid.classList.add('grid-container');
+
+const Footer = document.querySelector('.footer');
+body.insertBefore(Grid, Footer);
+
+for (let i=0; i<Cards.length; i++)
+{
+  const card = document.createElement('div');   
+  card.classList.add(Cards[i].class);  
+  Grid.appendChild(card);     
+  card.innerHTML=`
+  <img src="Images/Img_Placeholder.png" alt="#" class="my-project-image">
+  <div id="content">
+  <h3 class="my-work-title">${Cards[i].title}</h3>    
+  <p class="my-work-description">${Cards[i].description}</p>
+  <ul class="my-work-tools">
+    <li>${Cards[i].tags[0]}</li>          
+    <li>${Cards[i].tags[1]}</li>          
+    <li>${Cards[i].tags[2]}</li>          
+    <li>${Cards[i].tags[3]}</li>
+  </ul>
+  <button class="visualize-button" type="button">See Project</button>
+  </div> 
+  `;
+}
+
+const open = document.querySelectorAll('.visualize-button');
+for (let i = 0; i < Cards.length; i ++) {
+open[i].addEventListener('click', () => {
+const detail = document.createElement('div');
+detail.classList.add('details'); 
+body.appendChild(detail); 
+detail.innerHTML=`
+<div id="head">
+<h2>${Cards[i].title}</h2>
+<img id="close" src="Images/x.svg" alt="Close">
+</div>
+<img src="Images/Snapshoot_Portfolio.png" id="Pop_Img" alt="Image description">
+<p>${Cards[i].description}</p>
+<div id="Cwt">
+  <ul class="my-work-tools">
+    <li>${Cards[i].tags[0]}</li>          
+    <li>${Cards[i].tags[1]}</li>          
+    <li>${Cards[i].tags[2]}</li>          
+    <li>${Cards[i].tags[3]}</li>
+  </ul>  
+</div>
+<div id="Buttons">
+  <button class="vbutton-live" type="button"><a href="${Cards[i].link}">See Live</a></button>
+  <button class="vbutton-source" type="button"><a href="${Cards[i].source}">See Source</a></button>
+</div>
+`;
+body.appendChild(detail);
+
+document.querySelector('.nav-bar').style.display = 'none';
+
+  const close = document.getElementById('close');
+  close.addEventListener('click', () => {
+  body.removeChild(detail);
+  document.querySelector('.nav-bar').style.display = 'flex';
+  });
+});
+}
+
```

Comment:
- [ ] Great job by implementing your popup, but I think you still have some issues to fix about the design, your design should stick as much as possible to the Figma design that you chose, and also the container of your popup need to be visible :grin: :grin: 

Your design 
![Screenshot from 2022-07-23 00-28-01](https://user-images.githubusercontent.com/74501523/180577131-7326f4dc-389b-43a2-b9c2-dc55b06b556f.png) | 
![Screenshot from 2022-07-23 00-27-25](https://user-images.githubusercontent.com/74501523/180577151-682cbb26-dfcb-4899-b1d5-015fc8371b40.png)

Figma design
![Screenshot from 2022-07-23 00-31-06](https://user-images.githubusercontent.com/74501523/180577207-2944445c-fb1c-4ddb-9c0d-8938471dd7c1.png)
![Screenshot from 2022-07-23 00-31-44](https://user-images.githubusercontent.com/74501523/180577231-500688b9-9a77-45c3-afcc-960cb95f5c2c.png)

Reply:
Good evening!
Thank you so much for your feedback!

In figmas template, there's nothing behind it because it's just a template but the container is not solid, you can see the body background behind it.